### PR TITLE
Jdp210201 1

### PR DESCRIPTION
**Pull request by Bauaser-kun**

### CODE REVIEW

**Review by Ktubbo — DISMISSED** [review]

**Review by Ktubbo — CHANGES_REQUESTED** [review]
Edit (01.03): I think that you should not push Product class, because @Maciek2k3 has made it properly in _Product entity_ branch.

File: src/main/java/com/kodilla/ecommercee/ProductController.java [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+
+    @PutMapping(value = "updateProduct")
+    public ProductDto updateProduct(@RequestBody ProductDto productDto) {
+        return new ProductDto (1, "edited sample product", "edited sample description", 2.5, 1);
+    }
+
+    @DeleteMapping(value = "deleteProduct")
+    public void deleteProduct() {
+
+    }
```
Comment (Maciek2k3) [inline src/main/java/com/kodilla/ecommercee/ProductController.java]: I know that is void, but don't we should add @RequestParam with Id to @DeleteMappling.

**Review by Maciek2k3 — COMMENTED** [review]
https://guides.github.com/features/mastering-markdown/

File: src/main/java/com/kodilla/ecommercee/ProductController.java (hunk 1 above)
Comment (wisniewskimateusz) [reply src/main/java/com/kodilla/ecommercee/ProductController.java]: yes, (**@RequestParam Long id**)

**Review by wisniewskimateusz — DISMISSED** [review]
1. In deleteProduct should by (@RequestParam Long id)
2. In ProductDao int id -> Long id

**Review by Ktubbo — APPROVED** [review]

**Review by Maciek2k3 — APPROVED** [review]

**Pull request merged**